mas/db/ingest_pipeline.sh

```diff
@@ -0,0 +1,140 @@
+set -u
+set -e
+which concurrent
+set +e
```

Comment:
Perhaps remove 'set +e'? Running the whole script with `-e` is usually a good idea.

Reply:
Also, a bunch of warnings from `mdl`, the Markdown linter. You can install it yourself using `gem` (it's written in Ruby):

```
./mas/README.md:7: MD009 Trailing spaces
./mas/README.md:1: MD022 Headers should be surrounded by blank lines
./mas/README.md:5: MD022 Headers should be surrounded by blank lines
./mas/README.md:12: MD022 Headers should be surrounded by blank lines
./mas/README.md:15: MD031 Fenced code blocks should be surrounded by blank lines
./mas/README.md:17: MD031 Fenced code blocks should be surrounded by blank lines
./mas/README.md:8: MD032 Lists should be surrounded by blank lines
./mas/README.md:18: MD032 Lists should be surrounded by blank lines
./mas/MAS_Design.md:170: MD022 Headers should be surrounded by blank lines
./mas/MAS_Design.md:244: MD022 Headers should be surrounded by blank lines
./crawl/README.md:23: MD003 Header style
./crawl/README.md:10: MD009 Trailing spaces
./crawl/README.md:20: MD009 Trailing spaces
./crawl/README.md:1: MD022 Headers should be surrounded by blank lines
./crawl/README.md:12: MD022 Headers should be surrounded by blank lines
./crawl/README.md:23: MD022 Headers should be surrounded by blank lines
./crawl/README.md:27: MD022 Headers should be surrounded by blank lines
./crawl/README.md:23: MD026 Trailing punctuation in header
./crawl/README.md:15: MD031 Fenced code blocks should be surrounded by blank lines
./crawl/README.md:6: MD032 Lists should be surrounded by blank lines
./crawl/README.md:24: MD032 Lists should be surrounded by blank lines
```

Reply:
I added double quotes if the variables are strings. For integers, I leave them no quotes.

Reply:
I never imagine markdown has linter too :+1: 

"Perhaps remove 'set +e'? Running the whole script with -e is usually a good idea." - For the subsequent bash code, there are some errors i need to ignore for the ingestion to complete. So I need set +e in this case. If the errors are fatal, I explicitly check for them.